src/main/java/org/apache/hadoop/hdfs/server/namenode/TransferFsImageWrapper.java

```diff
@@ -46,21 +46,21 @@
  */
 public class TransferFsImageWrapper {
 
-  private final NNLoader nnLoader;
+  private final NameNodeLoader nameNodeLoader;
```

Comment:
Maybe stick with the `loader` variable name here too?

Reply:
Not sure how that sucker slipped by!